src/control/security/grpc_certs.go

```diff
@@ -44,9 +44,16 @@ func GetServerTransportCredentials(cfg *TransportConfig) (credentials.TransportC
 	}
 
 	tlsConfig := tls.Config{
-		ClientAuth:   tls.RequireAndVerifyClientCert,
-		Certificates: []tls.Certificate{*cfg.tlsKeypair},
-		ClientCAs:    cfg.caPool,
+		ClientAuth:               tls.RequireAndVerifyClientCert,
+		Certificates:             []tls.Certificate{*cfg.tlsKeypair},
+		ClientCAs:                cfg.caPool,
+		MinVersion:               tls.VersionTLS12,
+		MaxVersion:               tls.VersionTLS12,
+		PreferServerCipherSuites: true,
+		CipherSuites: []uint16{
+			tls.TLS_RSA_WITH_AES_128_GCM_SHA256,
+			tls.TLS_RSA_WITH_AES_256_GCM_SHA384,
```

Comment:
Why these suites, specifically? A comment with the rationale would be helpful, especially down the road when this may need to be updated.

Reply:
They are the SAFE approved guidance for cipher suites.